pkg/machine/qemu/machine.go

```diff
@@ -318,25 +318,28 @@ func (v *MachineVM) Init(opts machine.InitOptions) (bool, error) {
 		source := paths[0]
 		target := source
 		readonly := false
+		security_model := "mapped-xattr"
 		if len(paths) > 1 {
 			target = paths[1]
 		}
 		if len(paths) > 2 {
 			options := paths[2]
 			volopts := strings.Split(options, ",")
 			for _, o := range volopts {
-				switch o {
-				case "rw":
+				switch {
+				case o == "rw":
```

Comment:
does altering this switch really impact anything?

Reply:
This was necessary to avoid a compilation error:
```
invalid case strings.HasPrefix(o, "security_model=") in switch on o (mismatched types bool and string)
```

Compare a working mock-up:
https://go.dev/play/p/U4c9fYNngrp
...to a failing one:
https://go.dev/play/p/Y7CaQFOjrH_J

If there's a better way, please let me know. This is the first go code I've ever written.